src/main/java/edu/depaul/cdm/se452/d2l_mock/user_permission/UserService.java

```diff
@@ -0,0 +1,24 @@
+package edu.depaul.cdm.se452.d2l_mock.user_permission;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
```

Comment:
Do we need this class for MS2? Whats the difference between this and `User`?

Reply:
User represents a user where UserService checks to see whether or not they are a student or professor for permissions access.